Tests/Marketplace/prepare_private_id_set_for_merge.py

```diff
@@ -0,0 +1,37 @@
+import json
+import argparse
+from Tests.Marketplace.download_private_id_set import download_private_id_set_from_gcp
+
+
+def remove_old_pack_from_private_id_set(private_id_set_path, new_pack_name):
+    with open(private_id_set_path, 'r') as id_set_file:
+        private_id_set = json.load(id_set_file)
+
+    for entity, entity_list in private_id_set.items():
+        for item in entity_list[:]:
```

Comment:
why not `for item in entity_list:` ?

Reply:
because remove function jumps on the index of the list